rust/sdk-processor/src/steps/default_processor/default_extractor.rs

```diff
@@ -48,13 +47,15 @@ impl Processable for DefaultExtractor {
         ProcessorError,
     > {
         let flags = self.deprecated_table_flags;
-        let (block_metadata_transactions, table_items, current_table_items, table_metadata) =
+        let (block_metadata_transactions, raw_table_items, current_table_items, table_metadata) =
             process_transactions(transactions.data, flags);
+        let postgres_table_items: Vec<TableItem> =
```

Comment:
why did we need this change?

Reply:
@yuunlimm might be helpful for reviewers to update the PR description about the new format for models that are shared between Postgres and Parquet

Reply:
sorry missed updating the description! added little context in the pR description. @dermanyang let me know if this helps